lib/src/virtual_host.dart

```diff
@@ -44,23 +44,23 @@ abstract class VirtualHost {
 }
 
 class _VirtualHostDomain {
-  StreamController<HttpRequest> any;
-  StreamController<HttpRequest> exact;
+  StreamController<HttpRequest>? any;
+  StreamController<HttpRequest>? exact;
   Map<String, _VirtualHostDomain> subDomains = {};
 }
 
 class _VirtualHost implements VirtualHost {
   final _VirtualHostDomain _topDomain = _VirtualHostDomain();
-  StreamController<HttpRequest> _unhandledController;
+  StreamController<HttpRequest>? _unhandledController;
 
   @override
   Stream<HttpRequest> get unhandled {
     _unhandledController ??= StreamController<HttpRequest>();
 
-    return _unhandledController.stream;
+    return _unhandledController!.stream;
```

Comment:
with the change above this whole getter just becomes `Stream<HttpRequest> get unhandled => _unhandledController.stream;`

Reply:
If you look at `_unhandled`, it changes flow control based on the existence of the null controller.

```dart
  void _unhandled(HttpRequest request) {
    if (_unhandledController != null) {
      _unhandledController!.add(request);
      return;
    }
    request.response.statusCode = HttpStatus.forbidden;
    request.response.close();
  }
```
How do I square that once I make that property late, since it's no longer nullable at that point?

Reply:
Yep resolving these I didn't catch that it was using the nullability aspect